integreat_cms/xliff/utils.py

```diff
@@ -145,16 +149,20 @@ def page_to_xliff(page, target_language, dir_name, only_public=False):
     if not source_translation:
         source_language = page.region.get_source_language(target_language.slug)
         logger.warning(
-            "Page translation %r does not have a source translation in %r and therefore cannot be exported to XLIFF.",
+            "Page translation %r does not have a %ssource translation in %r and therefore cannot be exported to XLIFF.",
```

Comment:
No, since the interpolated string can also be an empty string, an additional space would mean two spaces in this case... Looks a bit ugly like this but I see no good way around it :see_no_evil: